pkg/resource/alias/sdk.go

```diff
@@ -76,6 +78,18 @@ func (rm *resourceManager) sdkFind(
 	// Merge in the information we read from the API call above to the copy of
 	// the original Kubernetes object we passed to the function
 	ko := r.ko.DeepCopy()
+	// Filter resulting aliases, matching only the one with the name in the spec
+	matchingAliases := []*svcsdk.AliasListEntry{}
+	for _, elem := range resp.Aliases {
+		if elem.AliasName == nil || r.ko.Spec.Name == nil {
```

Comment:
`can r.ko.Spec.Name` check be placed outside the loop so that unnecessary iteration over resp.Aliases in not needed?